### PR TITLE
Add Scout & Colour Swatches

### PR DESCRIPTION
**Pull request by Rias500**
- Scout can be used to replace algolia
- Colour Swatches is a port of the Craft 2 Color Swatches plugin

### CODE REVIEW

File: config/craft2-plugins.php [hunk 1]
```diff
@@ -43,6 +48,11 @@
         'statusColor' => 'red',
         'status' => 'Discontinued (use [Simple Text](https://github.com/craftcms/simple-text) instead)'
     ],
+    'Color Swatches' => [
```
Comment (brandonkelly) [inline config/craft2-plugins.php]: Should be `ColorSwatches` (no space)

File: config/craft2-plugins.php [hunk 2]
```diff
@@ -29,6 +29,11 @@
         'statusColor' => 'orange',
         'status' => 'Currently in development'
     ],
+    'Algolia' => [
+        'handle' => 'algolia',
```
Comment (brandonkelly) [inline config/craft2-plugins.php]: `handle` should only be specified if you are saying that the exact same plugin has a new handle.

File: config/craft2-plugins.php [hunk 3]
```diff
@@ -43,6 +48,11 @@
         'statusColor' => 'red',
         'status' => 'Discontinued (use [Simple Text](https://github.com/craftcms/simple-text) instead)'
     ],
+    'Color Swatches' => [
+        'handle' => 'colorswatches',
```
Comment (brandonkelly) [inline config/craft2-plugins.php]: `handle` should only be specified if you are saying that the exact same plugin has a new handle.

**Comment by brandonkelly** [comment]
You also may want to duplicate the `Algolia` array for `SearchPlus`.

**Comment by Rias500** [comment]
SearchPlus is already in there, but I've made the requested changes!

**Comment by brandonkelly** [comment]
Thanks!

**Pull request merged**